docs/root/development/tools/intellij.rst

```diff
@@ -10,10 +10,9 @@ Using IntelliJ with Envoy Mobile
 
 To get started using IntelliJ with Envoy Mobile:
 
-1. Locally install `Bazel <https://docs.bazel.build/versions/master/install-os-x.html#install-on-mac-os-x-homebrew>`_
-2. Download a supported `IntelliJ version <https://www.jetbrains.com/idea/download/other.html>`_ supported by the Bazel plugin
-3. Apply local hacks to make IntelliJ work using the branch `hack-for-intellij <https://github.com/lyft/envoy-mobile/tree/hack-for-intellij>`_
-4. Open up the Envoy Mobile project using the Bazel import project wizard
+1. Download a supported `IntelliJ version <https://www.jetbrains.com/idea/download/other.html>`_ supported by the Bazel plugin
+2. Apply local hacks to make IntelliJ work using the branch `hack-for-intellij <https://github.com/lyft/envoy-mobile/tree/hack-for-intellij>`_
```

Comment:
It was wrong from before but the repo url is https://github.com/envoyproxy/envoy-mobile

Reply:
https://github.com/envoyproxy/envoy-mobile/pull/1957